packages/glimmer-runtime/lib/builder.ts

```diff
@@ -279,6 +279,14 @@ export class ElementStack {
     return fragmentBounds;
   }
 
+  insertTextBefore(nextSibling: Node, text: string): Fragment {
```

Comment:
nvm, I see why this is needed now. we need to insert a stable bounds (`Fragment`) here because the siblings etc might want to reference that